app/menus/DocumentMenu.tsx

```diff
@@ -327,6 +347,22 @@ function DocumentMenu({
               visible: !document.isStarred && !!can.star,
               icon: <StarredIcon />,
             },
+            {
```

Comment:
I know this is a bit confusing at the moment, this part of the codebase is kind of in permanent flux with the switch over to action definitions. Here you can use `actionToMenuItem` (see L367) to convert the action into a menu item without having to redefine the label/icon etc.

Reply:
Excellent! Thanks for the suggestion